model/metrics/container.yaml

```diff
@@ -14,6 +14,20 @@ groups:
         brief: "The CPU state for this data point. A container SHOULD be characterized _either_ by data points with no `state` labels, _or only_ data points with `state` labels."
         requirement_level: opt_in
 
+  - id: metric.container.cpu.usage
+    type: metric
+    metric_name: container.cpu.usage
+    stability: experimental
+    brief: "CPU usage"
+    note: >
+      CPU usage of the specific container on all available CPU cores, averaged over the sample window
```

Comment:
is it normalized per CPU or sum across all of them?

Reply:
[kubeletstats](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.102.0/receiver/kubeletstatsreceiver/documentation.md#containercpuusage) reports the total on all available CPUs, does not break down per cpu.
[dockerstats](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/dockerstatsreceiver/documentation.md) does the same but maybe in the future it can become more granular since [dockerAPI](https://docs.docker.com/engine/api/v1.45/#tag/Container/operation/ContainerStats) can provide per cpu metrics. In any case, if we'd like to change it the future we can add the `cpu.logical_number` attribute as `opt-in` similarly to [`cpu.state`](https://github.com/open-telemetry/semantic-conventions/pull/1128#discussion_r1634449851)? (that was also discussed at https://github.com/open-telemetry/semantic-conventions/pull/282#issuecomment-1909118310).

Reply:
So if I have 10 CPUs that are busy 100% of time, would this metric show 100% (1) or 1000% (10) ? Could you please clarify the brief so it's clear to the future readers. See my other comment on the value range as well.

Reply:
In other places we call a similar metric `*.cpu.utilization`  and measure the normalized one (divided by cpu count) and in a [0, 1] range.  (see JVM, process, system metrics)

It'd be inconsistent if we had a different name and measured a sum across all CPUs here. 

We also report `*.cpu.count` in other places (as a metric, it has high cardinality and can't be used as an attribute).
The `cpu.time` and `cpu.count` metrics can be used together to calculate utilization. Utilization is still reported as a relatively cheap convenience.

By introducing non-normalized `cpu.usage` in addition to `cpu.time` here we provide no way for users to calculate normalized thing and create any alerts.

Reply:
> So, the suggestion:
>
> add container.cpu.logical_count or similar metric
> add container.cpu.utilization metric (in [0,1] range) instead of container.cpu.usage

It has been preferred to use the `container.cpu.usage` in `kubeletstats` receiver after a change at https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/25901.

As I also mention at https://github.com/open-telemetry/semantic-conventions/pull/1128#discussion_r1642699619 the `*utilization` metrics can be calculated in various different ways in K8s against various different limits. For sure we are planning to add those `*utilization` metrics as SemConvs as part of the https://github.com/open-telemetry/semantic-conventions/issues/1032. 
So I guess that `container.cpu.usage` can be added independently of the `*utilization` ones.